server/db.py

```diff
@@ -7,7 +7,13 @@
 """
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
-# To be replaced with the actual engine
-engine = create_engine("sqlite:///:memory:", echo=True)
+
+# Fill these fields with your local DB information
+username = ""
+password = ""
+port = ""
+db_name = ""
```

Comment:
You could use `os.environ` to import these from the .env file for the local repo. Easier to maintain and avoid accidentally pushing sensitive info

Reply:
Good point. Thank you